core/drivers/imx/mu/imx_mu_platform.h

```diff
@@ -9,12 +9,12 @@
 /*
  * Return the number of reception channels
  */
-unsigned int imx_mu_plat_get_rx_channel(void);
+unsigned int imx_mu_plat_get_rx_channel(vaddr_t addr);
```

Comment:
I'd prefer `base` to better describe this parameter.

Reply:
Ok